### ISSUE TITLE
[A11Y] Unable to upload screenshot using keyboard buttons

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3419]
**Browser / Version**: Firefox Nightly 81.0a1 (2020-07-27) 
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Using "Tab" key, navigate to the URL field and fill it in. 
3. Using "Tab" key, navigate to "Confirm" button.
4. Reach the "Upload image" section and try to upload an image using keyboard keys.
5. Observe behavior.

**Expected Behavior:**
Uploading an image can be performed.

**Actual Behavior:**
Uploading an image can't be performed.

**Notes:**
1. Screenshot attached.
![image](https://user-images.githubusercontent.com/12184325/88666157-5bd69100-d0e8-11ea-810a-eb62d1e40275.png)
2. According to "Inspector>Accessibility" tab, the  "Interactive elements must be focusable" (Read more here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard).
![image](https://user-images.githubusercontent.com/12184325/88683862-f214b200-d0fc-11ea-81e2-3e25056080ab.png)

**Watchers:**
@softvision-oana-arbuzov
@cipriansv

**Comment by softvision-oana-arbuzov** [comment]
I can confirm this is fixed.

Tested with:
Browser / Version: Firefox Nightly 82.0a1 (2020-09-16), Chrome 85.0.4183.102
Operating System: Windows 10 Pro
